Services/Transaction/Handler/TransactionPostHandler.php

```diff
@@ -189,8 +185,6 @@ private function handleEntity($transaction, CrudEntityInterface $entity)
         }
 
         $transaction->setMessages($messages);
-        $em = $this->registry->getManagerForClass(get_class($transaction));
-        $em->persist($transaction);
```

Comment:
It's not looking like the individual transactions in a collection are being written anymore. Only the transaction for the whole request.